common/setups/rasr/gmm_system.py

```diff
@@ -1393,8 +1393,9 @@ def run(self, steps: Union[List[str], RasrSteps]):
                 self.jobs[all_c]["costa"].update_rqmt("run", {"mem": 8, "time": 24})
 
         for trn_c in self.train_corpora:
+            # TODO: this will always overwrite base if we have multiple corpora
             self.store_allophones(trn_c)
```

Comment:
I think according to https://github.com/rwth-i6/i6_core/blob/f82690c6f4cf585154100221c916b82ba324fb29/meta/system.py#L228 the proper fix would be 
```suggestion
            self.store_allophones(source_corpus=trn_c, target_corpus=trn_c)
```

Reply:
I was not 100% sure if this has any other implications, as then the allophones are not registered for the eval corpora if not written into "base".

Reply:
Eval corpora usually don't need an allophone file, as we don't have alignments for them. This having said.. there might be implications for SAT recognition.

But anyway we need to consider then what happens when different train corpora have different allophone files..

Another suggestion would be to assert that all allophones are the same and then omit the loop(s) altogether and register only one output.?